hw_2/script.js

```diff
@@ -0,0 +1,39 @@
+function randn_bm() {
+    let u = 0,
+        v = 0;
+    while (u === 0) u = Math.random();
+    while (v === 0) v = Math.random();
+    return (Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2) | 0;
+}
+
+function fillAndGroupRandom(count, randomFunc) {
```

Comment:
This function return to us a new object which you generate from rundom function, the function name is a bit opaque.

Reply:
https://github.com/BlackTruth/javascript/blob/0f090223bfbe9aef482c5d350255c17995e68193/hw_2/script.js#L9
Renamed function.